scripts/run-migration.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)